dali/operators/reader/loader/webdataset/tar_utils.cc

```diff
@@ -97,11 +97,10 @@ static tartype_t kTarArchiveType = {LibtarOpenTarArchive, [](int) -> int { retur
 
 TarArchive::TarArchive(std::unique_ptr<FileStream> stream_)
     : stream_(std::move(stream_)),
-      archiveoffset_(0),
-      instance_handle_(Register(this)),
-      eof_(false) {
+      instance_handle_(Register(this)) {
   tar_open(ToTarHandle(&handle_), "", &kTarArchiveType, 0, instance_handle_, TAR_GNU);
   this->stream_->Seek(0);
+  eof_ = this->stream_->Size() == 0;
```

Comment:
With the `_` suffix indicating a field, the only reason to use `this->` would be to force the interpretation of the identifier as a dependent name (in a template) - but it's not relevant here.

Reply:
done

Reply:
Hmmm... not done? It's a nitpick, so it can stay as it is, but... not done?